server/api/projects/actions.py

```diff
@@ -126,3 +127,103 @@ def post(self, project_id):
             error_msg = f"Send message all - unhandled error: {str(e)}"
             current_app.logger.critical(error_msg)
             return {"Error": "Unable to send messages to mappers"}, 500
+
+
+class ProjectsActionsFeatureAPI(Resource):
+    @tm.pm_only(True)
+    @token_auth.login_required
+    def post(self, project_id):
+        """
+        Set a project as featured
+        ---
+        tags:
+            - projects
+        produces:
+            - application/json
+        parameters:
+            - in: header
+              name: Authorization
+              description: Base64 encoded session token
+              required: true
+              type: string
+              default: Token sessionTokenHere==
+            - name: project_id
+              in: path
+              description: The unique project ID
+              required: true
+              type: integer
+              default: 1
+        responses:
+            200:
+                description: Featured projects
+            400:
+                description: Bad request
+            403:
+                description: Forbidden
+            404:
+                description: Project not found
+            500:
+                description: Internal Server Error
+        """
+        try:
+            ProjectService.set_project_as_featured(project_id)
+            return {"Success": True}, 200
+        except NotFound:
+            return {"Error": "Project Not Found"}, 404
+        except ValueError as e:
+            error_msg = f"FeaturedProjects POST: {str(e)}"
+            return {"error": error_msg}, 400
+        except Exception as e:
+            error_msg = f"FeaturedProjects GET - unhandled error: {str(e)}"
```

Comment:
Let's not show too much information in our error messages @JorgeMartinezG - what do you think? Just using `str(e)` reveals too much about the backend